docs/content/Contribution guidelines/operations/versioned-things.md

```diff
@@ -0,0 +1,142 @@
+  1105](https://github.com/kubestellar/kubestellar/pull/1103), the
+  latest development in the release-0.7 line.
+
+## Core container image
+
+The core container runs the kcp server and the KubeStellar core
+controllers and initializes the kcp workspaces. It is built from some
+contents of [the KubeStellar GitHub
+repo](https://github.com/kubestellar/kubestellar), specifically
+excluding some that are only need by users outside this
+container. This image is normally built using the `make` target
+"kubestellar-image". This image is referenced from the core Helm
+chart, which can be used directly from the Helm repo and also is used
+by the `kubectl kubestellar deploy` command (which has a fixed image
+tag in its source, which is edited when it is time to advance that
+tag).
+
+Following are some recent tags and what they correspond to.
+
+- `release-0.9`, for the release 0.9 line of development; currently
+  this points to the same image as `v0.9.0`
+  but may move in the future.
+
+- `v0.9.0` = `v0.9.0-alpha.1.git1f0e2cc-clean`, for the v0.9.0 release.
+
+## KubeStellar core Helm chart
+
+This Helm chart describes a deployment of the KubeStellar core
+components (kcp server, KubeStellar controllers, initialization) as
+workload for a Kubernetes cluster. A Helm chart has both a version
+number for the chart itself and a version number for the "application"
+it installs. The `kubectl kubestellar deploy` command uses the
+adjacent copy of this Helm chart, and users can directly reference a
+copy in the Helm repo.
+
+Following are the versions of some recent editions of the chart.
+
+| Chart Version | App Version | What it is |
+| ------------- | ----------- | ---------- |
+|       1       |     v0.9.0  | KubeStellar release-0.9 branch |
+|     0.4.0     |     v0.4.0  | KubeStellar release-0.7 branch |
+
+## Brew repository
+
+The HomeBrew repository at `kubestellar/kubestellar` has two formulae.
+One called `kcp_cli` has the kubectl plugins of kcp. Release v0.11.0
+of kcp is always used. A formula called `kubestellar_cli` has contents
+that are needed by a user who deploys the KubeStellar core as workload
+in a Kubernetes cluster or another user of that deployment. This
+formula fetches and installs a `kubestellaruser` release archive from
+GitHub. Users invoke the `brew` command directly to install, upgrade,
+and remove these formulae.
+
+Following are the versions vailable.
+
+- `kcp_cli` `v0.11.0` for that release of kcp
+- `kubestellar_cli` `v0.9.0`, for that release of KubeStellar.
+
+## GitHub repos
+
+### kubestellar
+
+The [kubestellar](https://GitHub.com/kubestellar/kubestellar) repo is
+the main repo for KubeStellar. It holds the code and documentation and
+the source of the Helm chart. The following branches and tags are most
+interesting.
+
+- branch `main` is the main line of development.
+- branch `release-0.9` is for development along the line of the 0.9 release.
+- tag `v0.9.0` is release 0.9.0.
+
+### helm
```

Comment:
I would combine this with the 'Helm' section above.

Reply:
done